com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java

```diff
@@ -342,7 +342,7 @@ protected ModVisitor makeModVisitor(CGNode n, Collection<PointerKey> result, Poi
     return makeModVisitor(n, result, pa, h, false);
   }
 
-  protected ModVisitor<T,? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
+  protected ModVisitor<T, ? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
```

Comment:
Are you sure @khatchad removed that wildcard? The [link you provided](https://github.com/wala/WALA/pull/172/files#diff-409e334c66c9e19b63fba57e6cb66973R345) looks to me like it's *adding* wildcards in two places (lines 345 and 371), but not removing any. Or am I looking at the diff backwards?